packages/design-system-v2/src/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useCheckbox } from "@react-aria/checkbox";
+import { useToggleState } from "react-stately";
+import { useFocusRing } from "@react-aria/focus";
+
+import { CheckboxProps } from "./Checkbox.types";
+import { StyledCheckbox } from "./Checkbox.styles";
+
+function Checkbox(props: CheckboxProps) {
+  const { children, isDisabled, isIndeterminate } = props;
+  const state = useToggleState(props);
+  const ref = React.useRef(null);
+  const { inputProps } = useCheckbox(props, state, ref);
+  const { focusProps, isFocusVisible } = useFocusRing();
+  const id = `ads-checkbox-${props.value}`;
```

Comment:
this will not be unique, which might be a problem, yeah?

Reply:
This should be unique, checkbox with same value is not a real use case.

Reply:
Yes, that is what I'm saying in my above comment. the way you're generating id here, there's no guarantee that it will be unique. Should we worry about this? Or should we consider using something like nanoid?

Reply:
We don't need to worry about this,
Developers should not implement more than one checkbox with same value, so until and unless the values are unique this id will also be unique.

Reply:
Okay, just explicitly call this out in the prop doc string please!